app/jsx/components/MarqueeComp.jsx

```diff
@@ -48,9 +49,19 @@ class CarouselComp extends React.Component {
     return (
       <div className="c-marquee">
         <div className="c-marquee__carousel">
-          <div className="c-marquee__carousel-cell" style={{backgroundImage: "url('http://lorempixel.com/500/500/nature')"}}>
-            <h1>Carousel Cell Title 1</h1>
-            <p>{faker.fake("{{lorem.paragraphs}}")}</p>
+          <div className="c-marquee__carousel-cell" style={{backgroundImage: "url('"+ MEDIA_PATH + 'hero-ucb.jpg'+"')"}}>
```

Comment:
Sticking in a static image here simply to create the lightest possible background.